code/modules/clothing/clothing.dm

```diff
@@ -987,6 +987,45 @@
 		to_chat(usr, "<span class='notice'>You roll down your [src]'s sleeves.</span>")
 	update_clothing_icon()
 
+//Switch to taur sprites if a taur equips
+/obj/item/clothing/suit
+	var/taurized = FALSE
+
+/obj/item/clothing/suit/equipped(var/mob/user, var/slot)
+	var/normalize = TRUE
+
+	//Pyramid of doom-y. Improve somehow?

```

Comment:
> if(ishuman(user))
> > if((taurized && !isTaurTail(H.tail_style)) || (!taurized && isTaurTail(H.tail_style))
> > > taurize()

And then extract the actual sprite swapping stuff to a separate proc. I combined the if statements such that a single proc can be used to toggle it on and off, switching off of `taurized`